docs/machine-learning/how-to-guides/train-machine-learning-model-ml-net.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "How-To: Train a machine learning model in ML.NET" 
+description: Learn how to train various models in ML.NET
+ms.date: 04/09/2019
+author: luisquintanilla
+ms.author: luquinta
+ms.custom: mvc,how-to
+#Customer intent: As a developer I want to know how to train my model
+---
+
+# How-To: Learn how to train a machine learning model in ML.NET
+
+This how-to shows you how to build machine learning models, extract learned parameters and measure performance with ML.NET. Although this sample goes through the training of a regression model, the concepts are applicable throughout a majority of the other algorithms.
```

Comment:
Would this work? "Although this sample trains a regression model, the concepts are applicable throughout a majority of the other algorithms."